PROTOCOL.md

```diff
@@ -424,6 +430,49 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server. Most
+requests are sent with `REQUESTID`. `REQUESTID` is used to identify which
+action the request was intended for to protect from cases of trying to send
+multiple requests for the same action, which would otherwise effect future
+decisions. The first decision of a match has a `REQUESTID` of 1, and each
+successful decision increments the counter by 1.
+
+`|/team ODRER|REQUESTID`
+
+> Chooses the team order. Only as many Pokemon that need to be ordered need to be
+> specified: for singles only the lead Pokemon needs to be chosen; for doubles and
+> triples only two and three numbers need to be sent respectively; for teams with
+> Zoroark all six team members must be listed. As an example `/team 153246` in triples
+> will lead with the 1st team member in the leftmost slot, 5th in the middle, third
+> in rightmost slot, and 2nd, 4th, and 6th in reserve.
+
+`|/choose move NUMBER TARGET|REQUESTID` or `|/choose switch NUMBER|REQUESTID`
+
+> Requests to use the selected move `NUMBER` on `TARGET` Pokemon or to switch to
+> `NUMBER` Pokemon where `NUMBER` is the position in the team order. `NUMBER` is
+> 1-indexed, which means that `1` refers to the first move in the set. When a
```

Comment:
I'm not really feeling these first three lines, but I'm not sure how to explain everything without getting super wordy.